src/core/net_db.cc

```diff
@@ -108,7 +108,7 @@ void NetDb::Run() {
         m_Queue.GetNextWithTimeout(
             static_cast<uint16_t>(NetDbDuration::WaitForMessageTimeout));
       if (msg) {
-        int num_msgs = 0;
+        uint8_t num_msgs = 0;
```

Comment:
As mentioned in another review, we should use fully-qualified types (i.e., `std::uint8_t`).

Also, you'll want to work on `if (num_msgs > 100)` in terms of this PR refactor as these are both related.